js-exercises/duplicate-letters/duplicateLetters.js

```diff
@@ -0,0 +1,24 @@
+
+function duplicateLetters(args) {
+  let chars = {};
```

Comment:
A map would have been more useful here, as it has `get` and `set` methods already defined